client/components/NewMessage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import people from '../people.js'
+
+const styles = {
+  borderRadius: '150px',
+  background: 'white',
+  width: '300px',
+  height: '300px'
+}
+
+class NewMessage extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      sender: null,
+      recipient: null,
+      senderPhoto: null,
+      recipientPhoto: null
```

Comment:
You should add a `message` property here too since you create it when the image/message input box changes.